.github/workflows/distrib.yml

```diff
@@ -44,50 +44,36 @@ jobs:
   packaging-x86_64:
     # Run on push to the 'master' and release branches of tarantool/tarantool
     # or on pull request if the 'full-ci' or 'static-build-ci' label is set.
-    if: github.repository == 'tarantool/tarantool' &&
-        ( github.event_name != 'pull_request' ||
-          contains(github.event.pull_request.labels.*.name, 'full-ci') ||
-          contains(github.event.pull_request.labels.*.name, 'static-build-ci') )
+    if: github.repository == 'tarantool/tarantool'
```

Comment:
This 'ci: testing' commit is to verify the changes and it will be dropped before pushing the image?

It was not obvious on a glance. I usually add a `DO NOT MERGE:` prefix to such commits.

Reply:
> This 'ci: testing' commit is to verify the changes and it will be dropped before pushing the image?

Yep.